app/assets/javascripts/active_admin/lib/has_many.js.coffee

```diff
@@ -52,9 +52,10 @@ $ ->
   $(document).on 'change','.has_many_container[data-sortable] :input[name$="[_destroy]"]', ->
     recompute_positions $(@).closest '.has_many'
 
-  init_sortable()
   $(document).on 'has_many_add:after', '.has_many_container', init_sortable
 
+$(document).on 'ready page:load', ->
```

Comment:
Additionally, you could include it using the initial syntax, as in:

``` coffeescript
$(document).on 'ready page:load turbolinks:load', ->
  # ...
```

I'm impartial between the two.